src/sim-host/socket.js

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I created two new directories: `ui` and `protocol`. `socket.js` went in `protocol`, while all the user interface files went into `ui`.